<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordGenerator</class>
 <widget class="QMainWindow" name="PasswordGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>432</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>432</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Генератор паролей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>432</width>
     <height>245</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>432</width>
     <height>245</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="AllUI">
      <item>
       <layout class="QHBoxLayout" name="UserInput">
        <item>
         <layout class="QVBoxLayout" name="Settings">
          <item>
           <layout class="QVBoxLayout" name="CheckBoxes">
            <item>
             <widget class="QCheckBox" name="UppercaseLetters">
              <property name="text">
               <string>Прописные буквы (A - Z)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="Numbers">
              <property name="text">
               <string>Цифры (0 - 9)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="SpecialSymbols">
              <property name="text">
               <string>Специальные символы</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="Spaces">
              <property name="text">
               <string>Пробелы</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="PasswordSizeLayout">
            <item>
             <widget class="QLabel" name="PasswordSizeText">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Размер пароля</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="PasswordSize">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Buttons">
          <item>
           <widget class="QPushButton" name="Generate">
            <property name="minimumSize">
             <size>
              <width>245</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>245</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Сгенерировать</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="Options">
            <item>
             <widget class="QPushButton" name="Copy">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Копировать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Clear">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Отчистить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="BottomLayout">
        <item>
         <layout class="QHBoxLayout" name="GeneratedPasswordLayout">
          <item>
           <widget class="QLabel" name="GeneratedPasswordText">
            <property name="minimumSize">
             <size>
              <width>81</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>81</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="GeneratedPassword">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="WeakPasswodLayout">
          <item>
           <widget class="QLabel" name="WeakPasswordText">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Один из 200 ненадежных паролей:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="WeakPassword">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
